src/languages/es.js

```diff
@@ -680,6 +680,7 @@ export default {
             growlMessageOnSave: '¡La configuración del espacio de trabajo se ha guardado correctamente!',
             growlMessageOnDelete: 'Espacio de trabajo eliminado',
             deleteConfirmation: '¿Estás seguro de que quieres eliminar este espacio de trabajo?',
+            growlMessageOnDeleteError: 'No se puede eliminar el espacio de trabajo porque tiene informes que están siendo procesados.',
```

Comment:
I noticed that the english version does not have a period in the end. I think we should remove it here.

Reply:
+1!